### ISSUE TITLE
Infinite loop on follows when account reaches 7500 followings

### ISSUE THREAD

**Issue by manguszta** [instabot-py/instabot.py#2079]
Check this or we will delete your issue. (fill in the checkbox with an X like so: [x])
- [x] I have searched for other issues with the same problem or similar feature requests. 

#### Select one:
- [ ] Bug
- [x] Feature Request
- [ ] Technical Help
- [ ] Other

#### Environment
- [ ] Are you hosting the bot on a VPS or other Cloud hosting?
- [ ] If so, have you tried running the code on your local machine?

#### Operating System? (include version)
- [ ] macOS
- [x] Windows
- [ ] Linux (include flavour)
- [ ] Android (include terminal)

#### Python Version Requirement
- [x] I am using Python 3.6 or above

#### Description of your issue

```
I was wondering if there's any chance to examine whether my account reached the maximum 7500 followings? I'm getting in an infinite loop when only follows are in operation. The bot keeps trying to follow the same account over and over again, but it can't, because I reached 7500 followings. I'm only interested in a feature, maybe a condition that examines a request error from IG. I don't know, which error code is this, maybe an error 500? 
```

**Comment by waynoedm** [comment]
A feature would definitely be nice to throttle back the follows when account follow to unfollow ratio gets too high. I also have an account that ends up receiving a lot more follows from active, real, non-selebgram accounts than can be un-followed due to the criteria.  I have hit the 7500 mark on a couple of occasions and with the unfollow_everyone feature not working, I have to use third party apps to unfollow enough accounts to be able to run the bot effectively.

**Comment by sollertis8** [comment]
> A feature would definitely be nice to throttle back the follows when account follow to unfollow ratio gets too high. I also have an account that ends up receiving a lot more follows from active, real, non-selebgram accounts than can be un-followed due to the criteria. I have hit the 7500 mark on a couple of occasions and with the unfollow_everyone feature not working, I have to use third party apps to unfollow enough accounts to be able to run the bot effectively.

I agree, a follow throttling feature based on follow/unfollow ratio would be nice.  If you don't mind my asking, what third party apps are you using to unfollow accounts? The unfollowing rate in this bot doesn't adhere to my configuration settings.  It unfollows significantly less accounts than it follows regardless of settings and the amount of accounts available to unfollow.

**Comment by skvie** [comment]
I will check this soon

**Comment by skvie** [comment]
Fixed with https://github.com/instabot-py/instabot.py/commit/8d922986073fdc7442f8fd63cad2d31f88adb321
